ecology.yaml

```diff
@@ -0,0 +1,546 @@
+---
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: transformation
```

Comment:
Are those here part of W4M? Can you check the metabolomics yml file. If they are already included they are ok. ping @lecorguille 
It's hard for me to know which tool belongs to W4M or not :(

Reply:
I know ... :(
But it's too late to change that now.
New tools will be push as workflow4metabolomics.

Here is our group page within the TS: https://toolshed.g2.bx.psu.edu/groups#/175812cd7caaf439

Reply:
I trust @lecorguille very much. @yvanlebras feel free to add tools from this list that were updated in the last year.

Reply:
Ok, thank you Gildas, Björn. So I add the W4M trusted statistics related repos!